### PR TITLE
Add "Start cinema mod with Ctrl+Shift+Enter"

### LINKED ISSUE #9221: Cinema mod shortcut not implemented
**Describe the feature:** Shortcut is <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>Enter</kbd>. Right now it just enables autoplay (because of <kbd>Ctrl</kbd>+<kbd>Enter</kbd>) in song select.

### PR DESCRIPTION
**Pull request by orozso**
Add hotkey to "cinema" mod as in osu!stable.

Closes https://github.com/ppy/osu/issues/9221.

### CODE REVIEW

**Comment by bdach** [comment]
This has been attempted once before in #9224, and as far as I can see, [this implementation suffers from the exact same bug that prevented that one from being merged](https://github.com/ppy/osu/pull/9224#issuecomment-640112999).

**Review by bdach — CHANGES_REQUESTED** [review]
as above - please add automated test coverage for the aforementioned edge case if you decide to resolve

**Comment by peppy** [comment]
I'm not even sure we want this. Cinema is such a rare use case that I feel it probably doesn't need its own dedicated (hidden) hotkey..

File: osu.Game/Screens/Select/PlaySongSelect.cs [hunk 1]
```diff
@@ -108,5 +128,13 @@ protected override bool OnStart()
 
             return true;
         }
+
+        private Mod getActiveAutoMod()
+        {
+            if (Mods.Value.Any(m => m.GetType() == Ruleset.Value.CreateInstance().GetAutoplayMod()?.GetType())) return Ruleset.Value.CreateInstance().GetAutoplayMod();
+            else if (Mods.Value.Any(m => m.GetType() == Ruleset.Value.CreateInstance().GetCinemaMod()?.GetType())) return Ruleset.Value.CreateInstance().GetCinemaMod();
```
Comment (bdach) [inline osu.Game/Screens/Select/PlaySongSelect.cs:135]: I don't agree with this being handled locally as done here. The higher-level logic of mod selection/deselection should be deselecting one when the other is selected, via `IncompatibleMods` or similar. (see #7155) Otherwise we're going to end up with local fixes every time an issue like this arises.

That was also another problem the previous attempt at this had.

File: osu.Game/Screens/Select/PlaySongSelect.cs [hunk 2]
```diff
@@ -81,23 +92,32 @@ protected override bool OnStart()
...
 
-                var mods = Mods.Value;
-
-                if (autoType == null)
+                if (modType == null)
                 {
                     notifications?.Post(new SimpleNotification
                     {
-                        Text = "The current ruleset doesn't have an autoplay mod avalaible!"
+                        Text = "The current ruleset doesn't have an autoplay/cinema mod avalaible!"
```
Comment (bdach) [inline osu.Game/Screens/Select/PlaySongSelect.cs:102]: this text should depend on the hotkey pressed, not be an either/or catch-all

**Review by bdach — CHANGES_REQUESTED** [review]
